prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -108,21 +108,18 @@ class ReportFunction : Logging {
             PrincipalLevel.USER,
             workflowEngine
         )
-
-        // replace to make the senderName match with the ReportStream database (dashes instead of underscores)
-        // later, in the Okta Verifier,
-        // all the dashes in the sender.fullName are turned into underscores to match the Okta Organization naming scheme
-        val senderName = request.headers[CLIENT_PARAMETER]?.replace('_', '-')
```

Comment:
I noticed this code was redundant with a method extractClientHeader, so I'm using that now.